.editorconfig

```diff
@@ -18,3 +19,5 @@ dotnet_diagnostic.IDE0005.severity = warning
 # Sort system usings first
 dotnet_sort_system_directives_first = true
 
+[*.{json,csproj}]
+indent_size = 2
```

Comment:
Why go for an indent size of 4 for json and csproj for a smaller indent size?

Reply:
This is what I'm used to, but it is entirely a matter of taste. A lot of the dotnet projects, such as runtime and aspnetcore, use this convention.

Reply:
then let's go for it